scripts/www/build-cfg-help.pl

```diff
@@ -267,8 +270,28 @@ ()
         $data->{'default_doc'} = "";
         $data->{'default_if_none'} = "";
 
-        print "DEBUG: new option: $name\n" if $verbose;
+        print "DEBUG: line $linecount: new option: $name\n" if $verbose;
         next;
+    } elsif ($_ =~ /^IF (.*)$/) {
+        my $cond = $1;
+        push(@ifelse, $1);
+        if (! exists $defines{$1}) {
```

Comment:
It is very easy to accidentally create a hash entry in Perl. Where all hash entries must be defined, I recommend using `defined` instead of `exists` for testing:

```suggestion
        if (! defined $defines{$1}) {
```